boards/arm/stm32f0_disco/doc/stm32f0_disco.rst

```diff
@@ -0,0 +1,141 @@
+.. _stm32f0_disco_board:
+
+ST STM32F0DISCOVERY
+################
+
+Overview
+********
+
+The STM32F0 Discovery development board with STM32F051R8T6 MCU
+
+The STM32 Discoery board integrates the ST-LINK/V2-1 debugger and programmer.
+
+The STM32 Discovery board comes with the STM32 comprehensive software HAL library together
+with various packaged software examples.
+
+.. image:: img/en.stm32f0discovery.jpg
+     :width: 350px
+     :height: 404px
+     :align: center
+     :alt: STM32F0DISCOVERY
+
+More information about the board can be found at the `STM32F0DISCOVERY website`_.
+
+Hardware
+********
+
+- STM32F051R8T6 microcontroller featuring 64 KB Flash memory, 8 KB RAM in an LQFP64 package
+- On-board ST-LINK/V2 with selection mode switch to use the kit as a standalone ST-LINK/V2 (with SWD connector for programming and debugging)
+- Board power supply: through USB bus or from an external 5 V supply voltage
+- External application power supply: 3 V and 5 V
+- Four LEDs:
+    - LD1 (red) for 3.3 V power on
+    - LD2 (red/green) for USB communication
+    - LD3 (green) for PC9 output
+    - LD4 (blue) for PC8 output
+- Two push buttons (user and reset)
+- Extension header for all LQFP64 I/Os for quick connection to prototyping board and easy probing
+- An additional board is provided which can be connected to the extension connector for even easier prototyping and probing.
+- Comprehensive free software including a variety of examples, part of STM32CubeF0 package or STSW-STM32049 for legacy Standard Libraries usage
+
+More information about STM32F051R8 can be found here:
```

Comment:
Better to write this as:

    More information about STM32F051R8 can be found in the `STM32F0x8 reference manual`_.